roles/snapshot_sync/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Fetch the sui-tool binary
+  ansible.builtin.get_url:
+    url: "{{ sui[type].source_url }}/{{ sui[type].version }}/{{ snapshot_sync_tool_default_binary_name }}"
+    dest: "{{ snapshot_sync_bin_dir }}/{{ snapshot_sync_tool_default_binary_name }}"
+    owner: root
+    group: root
+    force: true
+    mode: "0755"
+
+- name: Stop service
+  ansible.builtin.systemd:
+    name: "{{ service_identifier }}"
+    state: stopped
+
+- name: Delete data folder directories
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: absent
+  loop:
+    - "{{ snapshot_sync_authorities_data_dir }}"
+    - "{{ snapshot_sync_consensus_data_dir }}"
+
+- name: Create data directories
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ service_identifier }}"
+    group: "{{ service_identifier }}"
+    mode: "0755"
+  loop:
+    - "{{ snapshot_sync_authorities_data_dir }}"
+    - "{{ snapshot_sync_authorities_data_dir }}/live"
+    - "{{ snapshot_sync_consensus_data_dir }}"
+
+- name: Download sui formal snapshot using sui-tool
+  ansible.builtin.command:
+    cmd: >
+      {{ snapshot_sync_bin_dir }}/{{ snapshot_sync_tool_default_binary_name }} download-formal-snapshot --latest
+      --genesis {{ snapshot_sync_genesis_file_path }}
+      --network {{ env }}
+      --path {{ snapshot_sync_authorities_data_dir }}/live
+      --num-parallel-downloads 50
```

Comment:
does it have to be "{{ snapshot_sync_authorities_data_dir }}/live" or just "{{ snapshot_sync_authorities_data_dir }}"

Reply:
Yes in their documentation they mentioned to resotre it to that directory

Reply:
https://docs.sui.io/guides/operator/snapshots#:~:text=When%20you%20restore,the%20path%20location.

Reply:
I think we still may just want to use /opt/sui/db/authorities_db and then have to rename the directory
<img width="1046" alt="Screenshot 2024-12-30 at 4 02 27 PM" src="https://github.com/user-attachments/assets/02685438-5a9e-423e-8c4d-2163133e587d" />